lib/services/db.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:hoopr/ballerCard.dart';
```

Comment:
Update this import to be `package:hoopr/models/user.dart` once that class is moved